ckan/config/middleware/flask_app.py

```diff
@@ -400,3 +405,7 @@ def is_blueprint(mm):
         for blueprint in inspect.getmembers(module, is_blueprint):
             app.register_blueprint(blueprint[1])
             log.debug(u'Registered core blueprint: {0!r}'.format(blueprint[0]))
+
+
+def _register_package_blueprints(app):
```

Comment:
can we name this method `_register_plugin_blueprints` ?

Reply:
Yep, i forgot that we'll need to extend this function with group blueprints

Reply:
thanks :)